RecoParticleFlow/PFTracking/src/PFDisplacedVertexFinder.cc

```diff
@@ -734,6 +732,17 @@ PFDisplacedVertexFinder::getTransvDiff(const GlobalPoint& Ref, const GlobalPoint
 
 }
 
+std::pair<float,float>
+PFDisplacedVertexFinder::getTransvLongDiff(const GlobalPoint& Ref, const GlobalPoint& ToProject) const {
+
+  Basic3DVector<float>vRef(Ref);
+  Basic3DVector<float>vToProject(ToProject);
+  float oneOverMag = 1.0f/vRef.mag();
+
+  return std::make_pair(fabs(vRef.cross(vToProject).mag()*oneOverMag),fabs((vRef.dot(vToProject)-vRef.mag2())*oneOverMag));
```

Comment:
it looks like the following should work
```Ref.basicVector.dot(ToProject.basicVector())```
same for .cross.
So, the conversion/copy is not really necessary.